jib-core/src/test/resources/sampleDockerfile

```diff
@@ -1,10 +1,10 @@
 FROM somebaseimage
 
-COPY libs /app/libs/
-COPY snapshot-libs /app/libs/
-COPY resources /app/resources/
-COPY classes /app/classes/
-COPY exploded-war /exploded/war/
+COPY libs /app/libs
```

Comment:
I think we actually need a trailing slash.

From https://docs.docker.com/engine/reference/builder/#copy:
> `If <dest> does not end with a trailing slash, it will be considered a regular file and the contents of <src> will be written at <dest>.`

Reply:
Oh, good catch.